flang/lib/Evaluate/intrinsics.cpp

```diff
@@ -1340,6 +1340,12 @@ static const IntrinsicInterface intrinsicSubroutine[]{
             {"values", AnyInt, Rank::vector, Optionality::optional,
                 common::Intent::Out}},
         {}, Rank::elemental, IntrinsicClass::impureSubroutine},
+    {"etime",
+        {{"values", TypePattern{RealType, KindCode::exactKind, 4}, Rank::vector,
+             Optionality::required, common::Intent::Out},
+            {"time", TypePattern{RealType, KindCode::exactKind, 4},
+                Rank::scalar, Optionality::required, common::Intent::Out}},
+        {}, Rank::elemental, IntrinsicClass::impureSubroutine},
```

Comment:
`etime` is obviously not elemental.

Reply:
That's something that had puzzled me too...  All the intrinsic subroutines do it that way, including `cpu_time` and `date_and_time`.  Not sure why that is, but when I started with this I came across it, too, and it got me wondering.

Reply:
It's just wrong, and should be fixed.

Reply:
Ok. But, please, separate PR and fix all of them in one go.  I think this is beyond what @JumpMasterJJ should do in here.

Reply:
Are you telling me to go fix them myself?

Reply:
@klausler Would this acceptable for you?

Reply:
Sounds good.